packages/dashboard/src/components/Wizard/PanelWizard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { IPanelWizardProps, IPanelWizardStyles, IPanelWizardStyleProps } from './PanelWizard.types';
+import { styled, IRenderFunction, classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { Panel, PanelType, IPanelProps } from 'office-ui-fabric-react/lib/Panel';
+import { Wizard } from './Wizard';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { defaultPanelStyleSet, defaultWizardStyleSet, getPanelWizardStyles } from './PanelWizard.styles';
+import { getStepContentToShow } from './Wizard.utils';
+
+const getClassNames = classNamesFunction<IPanelWizardStyleProps, IPanelWizardStyles>();
+
+/** Component for Setup Wizard */
+class PanelWizardBase extends React.Component<IPanelWizardProps, {}> {
+  public render(): JSX.Element {
+    if (this.props.wizardProps.steps.length === 0) {
+      throw new Error('Wizard must have atleast one step.');
+    }
+
+    const panelStyle = mergeStyleSets(
+      defaultPanelStyleSet(this.props.theme!),
+      this.props.panelProps ? this.props.panelProps.styles : undefined
+    );
+    const wizardStyle = mergeStyleSets(
+      defaultWizardStyleSet(this.props.theme!),
+      this.props.wizardProps.styles ? this.props.wizardProps.styles : undefined
+    );
+
+    return (
+      <Panel
+        type={PanelType.large}
+        onRenderNavigation={this._onRenderNavigationContent}
+        isFooterAtBottom={true}
+        onRenderFooter={this._onRenderFooter}

```

Comment:
>onRenderFooter={this._onRenderFooter} [](start = 8, length = 37)

Is the user going to provide the jsx for title and action bar?
Per design there is supposed to be back, next and save and close....

Does this mean, this has to be included by the consumer of the control?

Isn't it better to fix it here?

Reply:
I find some figma only have "next" and "exit", some of them have "back", "next" and "exit", some only have "done".

So I assumed it is a changing thing. By providing the element user also provides the onClick callbacks as they wish, like we discussed before.